packages/mail-client/src/mail-client.ts

```diff
@@ -87,25 +87,32 @@ function getCredentials(
   return { username, password };
 }
 
-async function createSocket(mailDestination: MailDestination): Promise<Socket> {
+/**
+ * @internal
```

Comment:
[pp] It would be helpful if there was a link to this `custom_auth_method: 0x80` thing so that someone can understand it immediately.